api/src/main/java/org/apache/iceberg/TableScan.java

```diff
@@ -94,6 +94,11 @@
    */
   TableScan includeColumnStats();
 
+  /**
+   * Doc doc doc
+   */
+  TableScan withManifestProcessor(ManifestProcessor processor);
```

Comment:
I see what you did here to get past the linter ;-)

If we decide to use this approach, let's be sure to not merge in `Doc doc doc` but instead a more descriptive comment or even a `TODO` comment. However since this is a WIP it's not worth your time to explain a pretty self documenting method name for the purpose of appeasing the linter. Leaving this comment so we don't forget to update if we do merge this in. :)

Reply:
Of course :)